### PR TITLE
Tests requests website

### PR DESCRIPTION
**Pull request by rjcf18**
PR for the tests requests website based on the issue tracker website

### CODE REVIEW

File: src/data/www/php71/dbox-tests/src/App/Instances/Model/InstanceModel.php [hunk 1]
```diff
@@ -140,14 +160,14 @@ public function add(array $src)
 	{
 		$filter = new InputFilter;
 
-		$instanceID = $src['php_version'] . $src['user_id'] . $src['pr_id'];
+		$instanceID = $src['php_version'] . 'u' . $src['user_id'] . 'p' . $src['pr_id'];
```
Comment (nibra) [inline src/data/www/php71/dbox-tests/src/App/Instances/Model/InstanceModel.php]: Why don't you just separate the values with dashes?

Comment (rjcf18) [reply src/data/www/php71/dbox-tests/src/App/Instances/Model/InstanceModel.php]: I thought this way the instance ID would be more descriptive, after the u user ID and after p PR ID but I can change to dashes as well it works with letters or dashes. Should I change to dashes then?

Comment (nibra) [reply src/data/www/php71/dbox-tests/src/App/Instances/Model/InstanceModel.php]: The question is: Does anybody benefit from that extra information? Dashes make a clearer separation optically, which IMO increasies readability.

Comment (rjcf18) [reply src/data/www/php71/dbox-tests/src/App/Instances/Model/InstanceModel.php]: Yeah you're right. I'll make that change then no prob ;)

**Pull request merged**